cmd/influx/main.go

```diff
@@ -425,16 +425,17 @@ func (c *CommandLine) parseInto(stmt string) string {
 }
 
 func (c *CommandLine) Insert(stmt string) error {
-	stmt = stmt[7:]
-	if strings.EqualFold(stmt[:4], "into") {
-		stmt = c.parseInto(stmt[5:])
+	i, point := parseNextIdentifier(stmt)
+	if !strings.EqualFold(i, "insert") {
+		fmt.Println("ERR: missing space after INSERT")
```

Comment:
I thought it would say something like "found INSERTxyx, expected INSERT".
That would be a more standard error message.

On Wednesday, June 24, 2015, Gunnar <notifications@github.com> wrote:

> In cmd/influx/main.go
> <https://github.com/influxdb/influxdb/pull/3082#discussion_r33171054>:
>
> > @@ -425,16 +425,17 @@ func (c *CommandLine) parseInto(stmt string) string {
> >  }
> >
> >  func (c *CommandLine) Insert(stmt string) error {
> > -	stmt = stmt[7:]
> > -	if strings.EqualFold(stmt[:4], "into") {
> > -		stmt = c.parseInto(stmt[5:])
> > +	i, point := parseNextIdentifier(stmt)
> > +	if !strings.EqualFold(i, "insert") {
> > +		fmt.Println("ERR: missing space after INSERT")
>
> For the line INSERTxyx, it prints out ERR: missing space after INSERT,
> tosses the line, and displays a new prompt. Would you expect the CLI to
> attempt to write point xyx?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdb/influxdb/pull/3082/files#r33171054>.
>